include/cantera/thermo/PengRobinson.h

```diff
@@ -0,0 +1,359 @@
+                              double w);
+
+    //! Set values for the interaction parameter between two species
+    /*!
+     *  The "a" parameter for interactions between species *i* and *j* is
+     *  assumed by default to be computed as:
+     *  \f[ a_{ij} = \sqrt(a_{i, 0} a_{j, 0}) + \sqrt(a_{i, 1} a_{j, 1}) T \f]
+     *
+     *  This function overrides the defaults with the specified parameters:
+     *  \f[ a_{ij} = a_{ij, 0} + a_{ij, 1} T \f]
+     *
+     *  @param species_i   Name of one species
+     *  @param species_j   Name of the other species
+     *  @param a0          constant term in the "a" expression [Pa-m^6/kmol^2]
+     *  @param a1          temperature-proportional term in the "a" expression
+     *      [Pa-m^6/kmol^2/K]
+     */
+    void setBinaryCoeffs(const std::string& species_i,
+                         const std::string& species_j, double a0, double a1);
+
+protected:
+    // Special functions inherited from MixtureFugacityTP
+    virtual double sresid() const;
+    virtual double hresid() const;
+
+	virtual double liquidVolEst(double TKelvin, double& pres) const;
+	virtual double densityCalc(double TKelvin, double pressure, int phase, double rhoguess);
+
+	virtual double densSpinodalLiquid() const;
+	virtual double densSpinodalGas() const;
+	virtual double pressureCalc(double TKelvin, double molarVol) const;
+	virtual double dpdVCalc(double TKelvin, double molarVol, double& presCalc) const;
+
+	// Special functions not inherited from MixtureFugacityTP
+
+	double daAlpha_dT() const;
+	double d2aAlpha_dT2() const;
+
+public:    
+
+    //! Calculate dpdV and dpdT at the current conditions
+    /*!
+     *  These are stored internally.
+     */
+    void pressureDerivatives() const;
+
+    //! Update the a and b parameters
+    /*!
+     *  The a and the b parameters depend on the mole fraction and the
+     *  temperature. This function updates the internal numbers based on the
+     *  state of the object.
+     */
+    virtual void updateMixingExpressions();
+
+    //! Calculate the a and the b parameters given the temperature
+    /*!
+     * This function doesn't change the internal state of the object, so it is a
+     * const function.  It does use the stored mole fractions in the object.
+     *
+     * @param temp  Temperature (TKelvin)
+     * @param aCalc (output)  Returns the a value
+     * @param bCalc (output)  Returns the b value.
+     */
+    void calculateAB(double& aCalc, double& bCalc, double& aAlpha) const;   
+
+    void calcCriticalConditions(double a, double b,double& pc, double& tc, double& vc) const;
+
+    //! Prepare variables and call the function to solve the cubic equation of state
+    int solveCubic(double T, double pres, double a, double b, double aAlpha,
+                     double Vroot[3]) const;
+protected:
+    //! Form of the temperature parameterization
+    /*!
+     *  0 = There is no temperature parameterization of a or b

```

Comment:
Is `b` allowed to be parameterized on the temperature? If not, can you remove it here?

Reply:
Also, this didn't quite turn into a bulleted list like I think you were thinking of.

Reply:
b is not temperature dependent, so I have removed it from the sentense.